contrib/collect-netscaler/version/version.go

```diff
@@ -0,0 +1,23 @@
+package version
```

Comment:
Don't duplicate the entire version.go here. I think it's fine to reuse the global one in internal/version just like collect-statse does. It isn't setting a precedent for code under contrib to use internal/* because internal/tsdb is already being used from there.

In general I think cmd/collect-statse belongs to contrib. It doesn't have any special status compared to collect-netscaler.